app/routers/models.py

```diff
@@ -98,14 +98,17 @@ class ParameterDecodedPublic(BaseModel):
     name: str
     type: str
     value: Any
-    value_decoded: (
-        Union[list["MultisendDecodedPublic"], "DataDecodedPublic", None] | None
-    ) = None
+    value_decoded: Union[
```

Comment:
We should use `|` instead `UNION` to express options.

Reply:
Agree, I used `Union` because previously it was not working for me